### DISCUSSION TITLE
Dark Mode (or Theme) via Vitepress / Tailwind

### DISCUSSION THREAD

**Discussion by phaufe** [ServiceStack/Discuss#3] [Ideas]
Optional dark mode as a theme for reading all the docs, changelogs, examples etc. Currently this is limited by Vitepress only having a general theme support (https://vitepress.vuejs.org/guide/theming.html) , and not an explicit dark mode as such. Maybe this will be extended though in some future versions.  Alternatively, Tailwind could maybe be used to add a dark mode.

More context for this, here is an issue on Vitepress about dark mode (https://github.com/vuejs/vitepress/issues/251) where someone actually found a solution that is already in use and working (https://github.com/vuejs/vitepress/issues/251#issuecomment-1020825712).

**Comment by mythz** [comment]
Happy to report that https://docs.servicestack.net now has Dark Mode thanks to migrating it to Razor SSG.

You can find information about the migration at: https://servicestack.net/posts/new_razor_ssg_docs
